web/src/components/molecules/Content/ContentTable/index.tsx

```diff
@@ -0,0 +1,72 @@
+import styled from "@emotion/styled";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import ConfigProvider from "@reearth-cms/components/atoms/ConfigProvider";
+import ProTable, { ListToolBarProps, ProColumns } from "@reearth-cms/components/atoms/ProTable";
+import { useT } from "@reearth-cms/i18n";
+
+const columns: ProColumns<any>[] = [];
+
+const ContentTable: React.FC = () => {
+  const dataSource: [] = [];
+  const t = useT();
+
+  const handleToolbarEvents: ListToolBarProps | undefined = {
+    search: {
+      onSearch: _ => {},
+    },
+  };
+
+  return (
+    <ConfigProvider
+      renderEmpty={() => (
+        <EmptyTableWrapper>
+          <Title>{t("No Items yet")}</Title>
+          <Suggestion>
+            {t("Create a new")}{" "}
+            <Button onClick={() => {}} type="primary">
+              {t("New Item")}
+            </Button>
+          </Suggestion>
+          <Suggestion>
+            {t("Or read")} <a href="">{t("how to use Re:Earth CMS")}</a> {t("first")}
+          </Suggestion>
+        </EmptyTableWrapper>
```

Comment:
Is there a redesign somewhere for this part? I can't find the design file that shows the Content page like this. In the design file, the New Item button is in the upper right corner with a plus icon.
Why I went looking for it was becuase in the screenshot the `create a new` and the `New Item` button look too close together

Reply:
@KaWaite , you are right, I was checking Figma now.
By default, I thought I have to show the content I put (50% of the system does).
But for this part, it's just an empty container if no data, I removed it.
Later I'll ask Red if we need to add it or not be the default (to make it a generic component).
![1](https://user-images.githubusercontent.com/78056580/193775304-6cd99783-2b83-4daa-8a2c-5489ea47a6ca.png)